slave.mk

```diff
@@ -638,15 +638,15 @@ ifeq ($(BLDENV),stretch)
 	STRETCH_DOCKER_IMAGES = $(filter $(SONIC_STRETCH_DOCKERS),$(DOCKER_IMAGES_FOR_INSTALLERS) $(EXTRA_DOCKER_TARGETS))
 	STRETCH_DBG_DOCKER_IMAGES = $(filter $(SONIC_STRETCH_DBG_DOCKERS), $(patsubst %.gz,%-$(DBG_IMAGE_MARK).gz, $(STRETCH_DOCKER_IMAGES)))
 else
-	DOCKER_IMAGES := $(SONIC_DOCKER_IMAGES)
-	DOCKER_DBG_IMAGES := $(SONIC_DOCKER_DBG_IMAGES)
+	DOCKER_IMAGES = $(filter-out $(SONIC_STRETCH_DOCKERS),$(SONIC_DOCKER_IMAGES))
+	DOCKER_DBG_IMAGES = $(filter-out $(SONIC_STRETCH_DBG_DOCKERS), $(patsubst %.gz,%-$(DBG_IMAGE_MARK).gz, $(STRETCH_DOCKER_IMAGES)))
```

Comment:
can you filter out both jessie docker and stretch docker here?

Reply:
Done.